src/Api/HealthChecks/DatabaseHealthCheck.cs

```diff
@@ -5,21 +5,20 @@ namespace Passwordless.Api.HealthChecks;
 
 public class DatabaseHealthCheck : IHealthCheck
 {
-    private readonly ITenantStorageFactory _tenantStorageFactory;
+    private readonly ITenantStorage _tenantStorage;
     private readonly ILogger<DatabaseHealthCheck> _logger;
 
-    public DatabaseHealthCheck(ITenantStorageFactory tenantStorageFactory, ILogger<DatabaseHealthCheck> logger)
+    public DatabaseHealthCheck(ITenantStorage tenantStorage, ILogger<DatabaseHealthCheck> logger)
     {
-        _tenantStorageFactory = tenantStorageFactory;
+        _tenantStorage = tenantStorage;
         _logger = logger;
     }
 
     public async Task<HealthCheckResult> CheckHealthAsync(HealthCheckContext context, CancellationToken cancellationToken = default)
     {
         try
         {
-            var storage = _tenantStorageFactory.Create(null);
-            await storage.GetAccountInformation();
+            await _tenantStorage.GetAccountInformation();
```

Comment:
@abergs Fixed it. It will also validate the connection to the database.

[Source](https://learn.microsoft.com/en-us/aspnet/core/host-and-deploy/health-checks?view=aspnetcore-8.0#entity-framework-core-dbcontext-probe)

I don't recall the option being there in .NET 6 or .NET 7.